### PR TITLE
Add persona.proto file

### LINKED ISSUE #1: Add Persona message to protobufs

### PR DESCRIPTION
**Pull request by joshx1**
Close #1 

Added persona.proto protobuf file to the centralized library of .proto files.

### CODE REVIEW

File: persona.proto [hunk 1]
```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+// This protobuf is for adding public/private key pairs to the messaging.
+message persona {
```
Comment (ahmedkadd) [inline persona.proto]: Capitalize Persona to follow the standard of other protobuf messages

File: persona.proto [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    message all_peers {
+        repeated Peer peer = 1;
+    }
+}
+
+message secret_book{
+    repeated persona person = 1;
+}
+
+message Peer {
```
Comment (ahmedkadd) [inline persona.proto]: A separate Peer protobuf message is not required if a boolean field is added to Persona indicating if it is self or peer

**Review by ahmedkadd — CHANGES_REQUESTED** [review]
The protobuf messages can be simplified. See my comment on how to replace the Peer message with a field in the Persona message.

File: persona.proto (hunk 1 above)
Comment (joshx1) [reply persona.proto]: Capitalized Persona.

File: persona.proto (hunk 2 above)
Comment (joshx1) [reply persona.proto]: Removed peer and added as boolean field to Persona.

**Review by ahmedkadd — APPROVED** [review]
All comments have been addressed. Approve.

**Pull request merged**